.travis.yml

```diff
@@ -12,6 +17,15 @@ install:
 script:
   - ./scripts/pre-commit.sh
   - vendor/github.com/kubernetes/repo-infra/verify/verify-boilerplate.sh -- rootdir="$GOPATH/src/github.com/kubernetes-sigs/federation-v2" -v
+deploy:
+  skip_cleanup: true
```

Comment:
Do we need `skip_cleanup` ?

Reply:
We need to ensure that the binaries built in the precommit remain available for the image build, so I think so.

Reply:
Yep, got it.